classes/Project/ImportProject.mjs

```diff
@@ -0,0 +1,142 @@
+import {Page} from "../Page/Page.mjs"
+import Project from "./Project.mjs"
+
+export default class ImportProject {
+  constructor(data) {
+    this.data = data
+  }
+
+  static async fetchManifest(manifestId) {
+    const url = `https://t-pen.org/TPEN/project/${manifestId}`
+    return fetch(url)
+      .then((response) => {
+        return response.json()
+      })
+      .catch((err) => {
+        return err
+      })
+  }
+
+  static async processManifest(manifest) {
+    let newProject = {}
+    newProject.title = manifest.label
+    newProject.label = manifest.label
+    newProject.metadata = manifest.metadata 
+    
+newProject["@context"]= manifest["@context"]
```

Comment:
duplicated line.  Also, it should be
```
newProject["@context"] = "http://t-pen.org/3/context.json"
newProject.manifest = manifest["@id"] ?? manifest.id
```

Make sure you are capturing the Manifest URI for the project.